### ISSUE TITLE
Share travis.yml among different api keys 

### ISSUE THREAD

**Issue by chandleryang76** [SDM2016AUT/SERLER_production#9]
I've thought of the way to share yml file with own heroku app.
How about the way below, 
-----------------------------
deploy:
  provider: heroku
  api_key:
    secure: ${API_KEY}
  app: ${APP_NAME}

--------------------------
We can make two variable api_key and name which are can set on each team's travis-ci repository.

**Comment by yuxiang8931** [comment]
deploy:
  provider: heroku
  api_key:
    team1_branch_name: team1's key
    team2_branch-name: team2's key
  app:
     team1_branch_name: app name
     team2_branch-name: app name

This might be the solution which we can use different branch name

**Comment by chandleryang76** [comment]
good idea, but I think If some other team update code, the code automatically deploy on other team's heroku app.

**Comment by yuxiang8931** [comment]
Just have a try tomorrow.

**Comment by chandleryang76** [comment]
Thank you for your efforts.

**Comment by yuxiang8931** [comment]
No problem, let's try tomorrow.
